index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="header">
+      Microverse Supermarket
```

Comment:
[OPTIONAL]  Kindly use a `h1` element for this line. 
This looks like the title of your project. For [semantic purposes](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html), it is advisable to use appropriate tags.